packages/bolt-connection/src/bolt/bolt-protocol-v5x8.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [https://neo4j.com]
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import BoltProtocolV5x7 from './bolt-protocol-v5x7'
+
+import transformersFactories from './bolt-protocol-v5x5.transformer'
+import Transformer from './transformer'
+
+import { internal } from 'neo4j-driver-core'
+
+const {
+  constants: { BOLT_PROTOCOL_V5_8 }
+} = internal
+
+export default class BoltProtocol extends BoltProtocolV5x7 {
+  get version () {
+    return BOLT_PROTOCOL_V5_8
+  }
+
+  get transformer () {
```

Comment:
JS noob question: Why is this override that's the exact same as its parent class necessary?